docs/UserGuide.md

```diff
@@ -176,7 +176,8 @@ Edits the details of a student.
 1. `edit_s` edits the student identified by the index. It only accepts a valid index
    based on the list when `list_s` is called.
 2. Tags are replaced if included in the argument, not appended.
```

Comment:
This one can remove if you included appending

Reply:
whoops, point number 2 can remove if you are intersecting all the tags

Reply:
The intersection is only used to check the presence of duplicated tags, the logic is still to replace the whole tag set. the only change is to add an additional warning about unchanged parts

Reply:
ok got it 👍🏻